arwen/elrondapi/elrondei.go

```diff
@@ -1361,6 +1366,42 @@ func writeLog(context unsafe.Pointer, pointer int32, length int32, topicPtr int3
 	output.WriteLog(runtime.GetSCAddress(), topics, log)
 }
 
+//export writeEventLog
+func writeEventLog(
+	context unsafe.Pointer,
+	numTopics int32,
+	topicLengthOffset int32,
```

Comment:
Suggest renaming to `topicLengthsOffset`, to emphasize that this is an array with multiple lengths.